public/app/features/admin/UserListPage.tsx

```diff
@@ -2,6 +2,8 @@ import { css } from '@emotion/css';
 import React, { useState } from 'react';
 
 import { GrafanaTheme2 } from '@grafana/data';
+import { selectors as e2eSelectors } from '@grafana/e2e-selectors/src';
+import { config, featureEnabled } from '@grafana/runtime/src';
```

Comment:
```suggestion
import { config, featureEnabled } from '@grafana/runtime';
```
Why are we importing from `src`?